### PR TITLE
Validate `status.election`.

### LINKED ISSUE #9: Certify votes
https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/blob/master/lib/worker.js#L309

1. TODO: send votes we know about to peer to optimize
2. TODO: validate `status.election`

Due to a glitch in the reporting tool, I'm not sure which of these two to-dos (or possibly both) we want to address.  Whatever the case, @dlongley, more explanation is needed here.

### PR DESCRIPTION
**Pull request by mattcollier**
Fixes #9.

### CODE REVIEW

File: lib/worker.js [hunk 1]
```diff
@@ -308,14 +309,16 @@ function _gossipWith(ledgerNode, voter, blockHeight, peer, callback) {
...
-      // TODO: validate `status.election`
       // TODO: make sure `Events` election topic is certified *BEFORE*
       //   roll-call because we depend on the votes for events to exist
       //   in the database (and they only get there if they are certified)
-      return async.eachSeries(status.election, (election, callback) => {
-        api._election.certify(
-          ledgerNode, election.topic, election.electionResults, callback);
-      }, callback);
+      return async.series([
+        callback => validate('continuity.election', status.election, callback),
```
Comment (mattcollier) [inline lib/worker.js]: @dlongley it was suggested that by adding validation to the client get block status API, that validation here may be unnecessary.  Is that true?  Does all the election information pass through that API?  Don't we have an issue for adding an API to send election data via a POST handler?

Comment (dlongley) [reply lib/worker.js]: Let's just leave it for now. We can look into optimizing it away later.

File: schemas/continuity.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+const schemas = require('bedrock-validation').schemas;
+
+const signature = {
```
Comment (dlongley) [inline schemas/continuity.js]: This exists:

https://github.com/digitalbazaar/bedrock-validation/blob/master/schemas/linkedDataSignature.js

Can we just reuse it? Maybe it needs an option for restricting the linked data signature type to just `LinkedDataSignature2015`. We could potentially just add a TODO to reuse it if that gets us there faster -- and can address later.

**Comment by mattcollier** [comment]
@dlongley revised the signature schema, ready for another pass.

**Pull request merged**